mindsdb/integrations/handlers/flaml_handler/flaml_handler.py

```diff
@@ -0,0 +1,47 @@
+import dill
+import pandas as pd
+from mindsdb.integrations.libs.base import BaseMLEngine
+from typing import Dict, Optional
+from type_infer.infer import infer_types
+from flaml import AutoML
+
+
+class FLAMLHandler(BaseMLEngine):
+    name = "FLAML"
+
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[Dict] = None) -> None:
+        if args is None:
+            args = {}
+
+        target_dtype = infer_types(df, 0).to_dict()["dtypes"][target]
+        model = AutoML(verbose=0)
+
+        if target_dtype in ['binary', 'categorical', 'tags']:
+            model.fit(X_train=df.drop(columns=[target]),
+                      y_train=df[target],
+                      task='classification',
+                      **args.get('using'))
+
+        elif target_dtype in ['integer', 'float', 'quantity']:
+            model.fit(X_train=df.drop(columns=[target]),
+                      y_train=df[target],
+                      task='regression',
+                      **args.get('using'))
+
+        if df is not None:
```

Comment:
When would this happen? You are using `df` before, in the `infer_types` call. Shouldn't this be checked at the very start, then?

Reply:
Sorry for this also 
i dont know how this missed my eyes